pkgs/applications/misc/input-leap/default.nix

```diff
@@ -64,6 +65,14 @@ stdenv.mkDerivation rec {
       libportal
     ];
 
+  patches = [
+    (fetchpatch2 {
+      # Upstream fix for crash on qt6.8 https://github.com/input-leap/input-leap/issues/2067
+      url = "https://github.com/input-leap/input-leap/commit/2641bc502e16b1fb7372b43e94d4b894cbc71279.patch";
+      hash = "sha256-QJtJ7USyc7FoVKSAOb0mlyqvxrGC6MPXvtyxpiyDAu8=";
```

Comment:
```suggestion
      url = "https://github.com/input-leap/input-leap/commit/2641bc502e16b1fb7372b43e94d4b894cbc71279.patch?full_index=1";
      hash = "";
```
Either add full_index=1 or use `fetchpatch`. Otherwise the hash may change.

Reply:
Hm, I didn't know this was possible. Would that only be in circumstances where a force push happens?

Reply:
No, the hash would change when the number of commits in the repository grows. If you don't add full_index=1 the patch file contains a shortened commit hash.

Reply:
I see, okay. That's interesting to know.